src/tools/cephfs/cephfs-shell

```diff
@@ -101,9 +102,29 @@ def get_chunks(file_size):
     yield(chunk_start, final_chunk_size)
 
 
-def to_bytes(string):
-    return bytes(string, encoding='utf-8')
-
+def to_bytes(param):
+    if isinstance(param, bytes):
+        return param
+    if isinstance(param, list):
+        return [i.encode('utf-8') for i in param]
+    elif isinstance(param, str):
+        return bytes(param, encoding='utf-8')
+    elif param is None:
+        return None
+    else:
+        return str(param).encode('utf-8')
+
+def to_str(param):
```

Comment:
This can be removed too. In most places bytes are directly decoded.

Reply:
What I want to get done is avoid this -

```
$ python3
Python 3.7.3 (default, Mar 27 2019, 13:41:07) 
[GCC 8.3.1 20190223 (Red Hat 8.3.1-2)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> a = [1, 2, 3]
>>> str(a)
'[1, 2, 3]'
>>> 
```

Reply:
It is not required if we keep the bytes list.